mavros/CMakeLists.txt

```diff
@@ -130,6 +130,7 @@ add_library(mavros_plugins
   src/plugins/hil_controls.cpp
   src/plugins/hil_actuator_controls.cpp
   src/plugins/home_position.cpp
+  src/plugins/setpoint_global_position.cpp
```

Comment:
Make sense, but keeping it separate might help with later support.
But perhaps with Geolib most code gone, so it is ok to extend sp-pos.